src/admin/components/ProCta.vue

```diff
@@ -75,26 +72,30 @@ export default {
     }
 
     .btn {
-        background-color: #fff;
-        color: #f2624d;
-        font-size: 16px;
+        background-color: #A244FF;
+        color: #FFFFFF;
+        font-size: 1.2rem;
         line-height: 1.538em;
         font-weight: 300;
-        border: 1px solid #FF9381;
-        border-radius: 3px;
-        box-shadow: 0px 1px 2px 0px rgba(255, 128, 142, 0.05);
-        display: inline-block;
+        border-radius: 14px;
         text-decoration: none;
-        padding: 12px 27px;
+        display: inline-flex;
+        padding: 16.05px 45.167px;
```

Comment:
Pls remove the fractional pixel.

Reply:
@jamil-mahmud bhai @tuhin480 bhai

<img width="1003" alt="Screenshot 2024-11-06 at 3 19 48 PM" src="https://github.com/user-attachments/assets/20904b6b-a5f5-464d-b2c1-d99a840c432c">